MetaGenerator.Test/UserInput/TimeRequesterTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using Funcky.Monads;
+using Messerli.CommandLineAbstractions;
+using Messerli.MetaGenerator.UserInput;
+using Messerli.MetaGeneratorAbstractions.Json;
+using Messerli.MetaGeneratorAbstractions.UserInput;
+using Moq;
+using Xunit;
+
+namespace Messerli.MetaGenerator.Test.UserInput
+{
+    public class TimeRequesterTest
+    {
+        public TimeRequesterTest()
+        {
+            CultureInfo.CurrentCulture = new CultureInfo("de-CH", false);
```

Comment:
Why don't you use a `TemporaryCultureSwitch`?

Reply:
Add and use TemporaryCultureSwich: 8f27d6a59a64bdc90f0b09f097d3f3e139a98379